src/main/java/com/microsoft/jenkins/kubernetes/KubernetesDeployContext.java

```diff
@@ -64,6 +70,24 @@
     private ConfigFileCredentials kubeConfig;
     private TextCredentials textCredentials;
 
+    private DeployTypeClass deployTypeClass;
+    private String deployType;
+    private HelmContext helmContext;
+    private String helmChartLocation;
```

Comment:
can below properties be  embedded into HelmContext?

Reply:
The properties are in the HelmContext, here is just for UI reflection.